packages/react-search-ui/src/containers/__tests__/Facet.test.js

```diff
@@ -187,6 +185,11 @@ describe("show more", () => {
   describe("after a show more click", () => {
     beforeAll(() => {
       wrapper.find(View).prop("onMoreClick")();
+
+      expect(params.a11yNotify).toHaveBeenCalledWith("moreFilters", {
```

Comment:
Yeah, it has to be. Because we have [a top-level `beforeEach`](https://github.com/elastic/search-ui/pull/359/files/da835da856df9c86641cd463a6c462803dcbf472#diff-324d2d9aaa8fcd2d17c5911cd7d3f082R50) that's clearing mocks on each new test, the a11yNotify spy gets cleared after this beforeAll. So if I moved this expect to another separate test it would fail.

We could opt to not clear the a11yNotify mock, but I don't like that very much (could lead to false positives). To me this test reads as "When you click the more button, an a11y notification is read out", and that's OK to me. If we want to make the test less busy I can swap `.toHaveBeenCalledWith()` for just `.toHaveBeenCalled()`, and thus we don't need to specify the counts or booleans or anything.

Reply:
Yes, totally agree now, thanks a lot for the explanation! I will definitely pick your brain when I write tests in the future, they're awesome in this PR 🙂